site/public/css/markdown.css

```diff
@@ -90,6 +92,89 @@
     padding: 3px;
 }
 
+/* Styling of markdown input space (MarkdownArea.twig)*/
+.markdown-area {
+    border-radius: 6px 6px 6px 6px;
+    border: 1px solid var(--standard-light-gray);
+}
+
+/* header (.markdown-area-header + children)*/
+.markdown-area-header {
+    display: flex;
+    flex-direction: row;
+    background-color: var(--standard-pale-gray);
+    padding: 0 10px 0 10px;
+    border-radius: 6px 6px 0 0;
+}
+
+[data-theme="dark"] .markdown-area-header {
+    background-color: var(--standard-deep-dark-gray);
+}
+
+.markdown-area-toolbar {
+    display: flex;
+    margin-left: auto;
+    align-items: center;
+}
+
+.markdown-mode-buttons {
+    display: flex;
+    margin-top: 10px;
+}
+
+.markdown-mode-tab {
+    font-size: 14px;
+    margin-left: 5px;
+    padding-left: 16px;
+    padding-right: 16px;
+    min-height: 35px;
+    cursor:pointer;
+}
+
+.markdown-mode-tab.active {
+    border: 1px solid var(--standard-light-gray);
+    border-radius: 6px 6px 0 0;
+    border-bottom-style: none;
+    background-color: var(--default-white);
+    color: var(--text-black);
+}
+
+.markdown-mode-tab:not(.active) {
+    border: 1px solid var(--standard-pale-gray);
+    background-color: var(--standard-pale-gray);
+    color: var(--standard-medium-gray);
+    transition: color 0.5s 0s;
+}
+
+[data-theme="dark"] .markdown-mode-tab:not(.active) {
+    background-color: var(--standard-deep-dark-gray);
+    border-color: var(--standard-deep-dark-gray);
+}
+
+.markdown-mode-tab:not(.active):hover {
+    color: var(--text-black);
+}
+
+
+.markdown-tool {
+    cursor: pointer;
+    margin-left: 3px;
+}
+
+/* body (.markdown-area-body + children) */
+.markdown-area-body {
+    padding: 16px;
+    background: var(--default-white);
+    border-radius: 0 0 6px 6px;
+}
+
+.markdown-area textarea {
+    resize: vertical;
+    min-height: 100px;
+    max-height: 300px;
+    width: -webkit-fill-available;
```

Comment:
That width attribute is actually redundant since the `markdown-area` element will always have the `fill-available` class on it which does the same thing and properly works for all browsers (has multiple fallbacks), so I've just removed the width style here.